src/components/ReviewForm/ReviewForm.jsx

```diff
@@ -0,0 +1,57 @@
+import { useCallback, useState } from 'react';
+
+import Rating from '@mui/material/Rating';
+import Typography from '@mui/material/Typography';
+
+import { AppButton } from '@/shared';
+import { AppTextArea } from '@/shared/AppTextArea/AppTextArea';
+import { ButtonWrapper, Form } from './ReviewForm.styled';
+
+export const ReviewForm = () => {
+  const [rating, setRating] = useState(0);
+  const [review, setReview] = useState('');
+
+  const handleTextareaChange = useCallback((e) => {
+    setReview(e.target.value);
+  }, []);
+
+  const handleFeedbackSubmit = (e) => {
+    e.preventDefault();
+  };
+
+  return (
+    <Form onSubmit={handleFeedbackSubmit} style={{ width: '600px' }}>
+      <label>
+        <Typography variant="h5" component="p" gutterBottom>
+          Rating
+        </Typography>
+        <Rating
+          name="simple-controlled"
+          value={rating}
+          onChange={(event, newValue) => {
+            setRating(newValue);
+          }}
+          sx={{ maxWidth: '205px', fontSize: '40px' }}
+        />
+      </label>
+
+      <label>
+        <Typography variant="h5" component="p" gutterBottom>
+          Review
+        </Typography>
+        <AppTextArea
+          minRows={5}
+          maxRows={5}
+          placeholder="Leave your feedback"
+          value={review}
+          onChange={handleTextareaChange}
+          maxLength={400}
+          name="review"
+        />
+      </label>
+      <ButtonWrapper>
+        <AppButton type="contained" label="Send" />
```

Comment:
I fixed the bug with AppButton props. So you can change it to the correct one <AppButton variant="contained" label="Send" />

Reply:
Hi. I changed it. Could you approve my PR? )